addons/stock/populate/stock.py

```diff
@@ -238,7 +238,6 @@ def _populate_factories(self):
             if products and products[0].type == 'product':
                 valid_product[suplierinfo.company_id.id] |= set(products.ids)
         valid_product = {company_id: product_ids | valid_product[False] for company_id, product_ids in valid_product.items() if company_id}
-        invalid_product = {company_id: list(all_product_ids - product_ids) for company_id, product_ids in valid_product.items() if company_id}
```

Comment:
How are these changes related to the bug you're trying to fix?

Reply:
The problem was that the population creates orderpoints with products that are consumable or don't have vendors (as an example of invalid orderpoints). Therefore, after enforcing the products to be storable, the runbot population build failed. That's why I removed invalid products from the population at the beginning. However, I changed it now and added the invalid products back again, however, only those without vendor. Non-storable products are excluded completely from the population.